gneiss/regression/_format.py

```diff
@@ -0,0 +1,31 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016--, gneiss development team.
+#
+# Distributed under the terms of the GPLv3 License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+import qiime2.plugin.model as model
+from gneiss.plugin_setup import plugin
+import pickle
+
+
+class RegressionFormat_g(model.BinaryFileFormat):
+    def sniff(self):
+        try:
+            # just check if the file is pickleable.
+            with open(str(self), 'rb') as fh:
+                pickle.load(fh)
```

Comment:
Yeah - I couldn't think of a better way to do it, since pickle files don't have headers. 
Do you have a better idea?